pkgs/top-level/python-packages.nix

```diff
@@ -1311,7 +1311,7 @@ in {
 
   canmatrix = callPackage ../development/python-modules/canmatrix {};
 
-  cairocffi = callPackage ../development/python-modules/cairocffi {};
+  inherit (callPackage ../development/python-modules/cairocffi {}) cairocffi cairocffi_0_9;
```

Comment:
How about
```suggestion
  cairocffi = if isPy3k then
    (callPackage ../development/python-modules/cairocffi {}).cairocffi
  else
    (callPackage ../development/python-modules/cairocffi {}).cairocffi_0_9;
```

Reply:
You're right! That made the fix for `thumbor` far easier. I also exposed `cairocffi_1_0` and `cairocffi_0_9` for now as it seems to be a common pattern to add all versioned attributes to the package set and alias one of those to the default package name.

Reply:
I'd prefer not to expose them because `pythonPackages` should only contain a single version of each package. Btw, I think the prettiest way to do this is by using `let inherit (calPackage ...) cairocffi cairocffi_0_9; in if isPy3k ...`.